src/basho_bench_worker.erl

```diff
@@ -56,20 +60,20 @@
 %% API
 %% ====================================================================
 
-start_link(SupChild, Id) ->
-    case basho_bench_config:get(distribute_work, false) of 
```

Comment:
Again, I think we have to be extremely careful about style changes, even though it's _normally_ the right thing to do. @chewbranca opinion?